gradle.properties

```diff
@@ -2,17 +2,17 @@
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 # FallingTree
-minecraft_version=1.18
+minecraft_version=1.18.1
 modId=editsign
 modName=EditSign
-version=2.2.6
+version=2.2.7
 releaseType=release
 # Repository
 repoUser=RakSrinaNa
 repoName=EditSign
 # Forge
 mappings_channel=official
-mappings_version=1.18
```

Comment:
This should be just `1.18.1`. 
What you changed here are the mappings for forge. This is probably why you didn't managed to get Forge working.